android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/AcctMgrFragment.java

```diff
@@ -82,12 +102,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         // Get the account managers list.
 		ArrayList<AccountManager> accountManagers = null;
-		if (mIsBound){
+		if (mIsBound && !mAdapterDataFilled){
```

Comment:
This looks redundant now. Could this if/else construct be added to and then replaced by the new `fillAdapterData()` method?